src/DoctrineBridge/Router/EntityIdResolver.php

```diff
@@ -25,11 +25,18 @@
     private $manager;
 
     /**
-     * @param ManagerRegistry $manager
+     * @var IdFromEntityInterface
```

Comment:
For now I will leave them in case I need to add scalar values. Once I can use proper typehints, I will omit docs were possible. I'm thinking of making 2.0 only available for php 7.0.